lib/oli_web/live/community_live/form.ex

```diff
@@ -15,7 +15,7 @@ defmodule OliWeb.CommunityLive.Form do
         {#if @display_labels}
           <Label class="control-label">Community Name</Label>
         {/if}
-        <TextInput class="form-control" opts={placeholder: "Name"}/>
+        <TextInput class="form-control" opts={placeholder: "Name", maxlength: "256"}/>
```

Comment:
Is the field UTF-8mb4 (if so this would be 191 instead of 256)?

Reply:
We're using UTF8 so we're ok, right?